optaplanner-persistence/optaplanner-persistence-jpa/src/test/filtered-resources/META-INF/persistence.xml

```diff
@@ -34,4 +34,37 @@
       <property name="hibernate.transaction.jta.platform" value="org.hibernate.service.jta.platform.internal.JBossStandAloneJtaPlatform" />
     </properties>
   </persistence-unit>
+
+  <persistence-unit name="org.optaplanner.persistence.jpa.converter.test" transaction-type="JTA">
+    <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
+    <jta-data-source>jdbc/testDS1</jta-data-source>
+
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.bendable.BendableScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.bendablebigdecimal.BendableBigDecimalScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.bendablelong.BendableLongScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.hardmediumsoft.HardMediumSoftScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.hardmediumsoftbigdecimal.HardMediumSoftBigDecimalScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.hardmediumsoftlong.HardMediumSoftLongScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.hardsoft.HardSoftScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.hardsoftbigdecimal.HardSoftBigDecimalScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.hardsoftdouble.HardSoftDoubleScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.hardsoftlong.HardSoftLongScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.simple.SimpleScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.simplebigdecimal.SimpleBigDecimalScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.simpledouble.SimpleDoubleScoreConverterTest$TestJpaEntity</class>
+    <class>org.optaplanner.persistence.jpa.impl.score.buildin.simplelong.SimpleLongScoreConverterTest$TestJpaEntity</class>
+    <exclude-unlisted-classes>true</exclude-unlisted-classes>
+
+    <properties>
+      <property name="hibernate.max_fetch_depth" value="3"/>
+      <property name="hibernate.hbm2ddl.auto" value="update" />
+      <property name="hibernate.show_sql" value="false" />
+
+      <property name="hibernate.dialect" value="${maven.hibernate.dialect}"/>
+      <property name="hibernate.default_schema" value="${maven.jdbc.schema}"/>
+
+      <!-- The following line is what's used in Hibernate 4 instead of a TransactionManagerLookup class -->
+      <property name="hibernate.transaction.jta.platform" value="org.hibernate.service.jta.platform.internal.JBossStandAloneJtaPlatform" />
+    </properties>
```

Comment:
Less is more. Less code is more maintenable. Why do we need all these extra lines?

Reply:
Hmm, I did this to separate the AttributeConverter tests from the HibernateType tests in a separate persistence unit, but it does result in a lot of duplication. Turns out this is not necessary! I'll refactor these tests.